pkg/proc/internal/ebpf/helpers.go

```diff
@@ -0,0 +1,190 @@
+//go:build ebpf
+// +build ebpf
+
+package ebpf
+
+// #include "./trace_probe/function_vals.bpf.h"
+import "C"
+import (
+	_ "embed"
+	"encoding/binary"
+	"errors"
+	"reflect"
+	"runtime"
+	"sync"
+	"unsafe"
+
+	"github.com/go-delve/delve/pkg/dwarf/godwarf"
+	"github.com/go-delve/delve/pkg/dwarf/op"
+
+	bpf "github.com/aquasecurity/libbpfgo"
+	"github.com/aquasecurity/libbpfgo/helpers"
+)
+
+//go:embed trace_probe/trace.o
+var TraceProbeBytes []byte
+
+const FakeAddressBase = 0xbeef000000000000
```

Comment:
For consistency with the other code this should be fakeAddressUnresolv (0xbeed000000000000) because we are using the 0xbeef000000000000 to describe registerized variables whose CompositeMemory is cached.

Reply:
Done.